console/Cargo.toml

```diff
@@ -50,6 +51,7 @@ test = [
 account = [ "network", "snarkvm-console-account" ]
 algorithms = [ "snarkvm-console-algorithms" ]
 collections = [ "algorithms", "snarkvm-console-collections" ]
+filesystem = [ "snarkvm-console-network/filesystem" ]
```

Comment:
Why not add an `sgx` feature that is not a default feature rather than injecting a `filesystem` feature that already represents the current behavior?

You could also enforce that the `target_env` must be `sgx` if the feature is enabled as well.

Reply:
I'm more in favour of `filesystem`, better not to have custom feature flags for every possible machine target (of which there are many, even differences between sgx machines)